CHANGELOG.md

```diff
@@ -20,6 +22,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - User experience for situations where Conjur identity cannot be resolved has been
   improved with better errors/warnings.
   [cyberark/conjur-puppet#125](https://github.com/cyberark/conjur-puppet/issues/125).
+- Added warnings and better error logic to certificate parsing logic.
```

Comment:
"This module now logs a warning if [XX certificate situation] and an error if [blah blah cert unreadable or whatever]"